Wordle.java

```diff
@@ -140,43 +146,29 @@ public static WordleSolver getSolver(Scanner in) throws Exception
 		}
 
 		// initialize the chosen solver
-		if (choice == 0)
-		{
-			NoDupesSolver solver = (NoDupesSolver)solvers[choice];
-			solver.initialize(loadWords(new File("words5.txt")), true);
-		}
-
-		return solvers[choice];
```

Comment:
Why did you remove this only to add the same thing but generalized so that all solvers have to take the same data? I chose to do it this way so that solvers that didn't need to do much work could get away with it. If we end up adding code that runs a bunch of simulated games, we want the solver initializations to be fast (no unnecessary file loading for unused parameters), or at least that was my logic.